app/components/about/about.html

```diff
@@ -5,10 +5,10 @@
 <p>
   You want more? Add them yourself!
 </p>
-<p>
+<form (submit)="addName(newname); $event.preventDefault();">
```

Comment:
I think `return false;` on the addName method should prevent submit default behaviour.
That would look a little bit prettier.